mvds.md

```diff
@@ -144,6 +145,15 @@ The record of the type `Type` SHOULD be retransmitted every time `Send Epoch` is
 1. <https://code.briarproject.org/briar/briar-spec/blob/master/protocols/BSP.md>
 2. <https://github.com/vacp2p/mvds>
 
+## Changelog
+
+| Version | Comment |
+| :-----: | ------- |
+| [0.6.0](https://github.com/vacp2p/specs/commit/b6335947e09502ec399954002d983a709065a342#diff-a6dbd535bfcb545a64b1c95f6dc8f9f1) | Update protocol buffer field numbers and other minor changes |
+| [0.5.2](https://github.com/vacp2p/specs/blob/0adddcdb8d7e71240c2df30d4ba40e49b4275011/mvds.md) | Update protocol buffer field numbers |
+| [0.5.1](https://github.com/status-im/bigbrother-specs/blob/20e2659447dcb5123a105526eb26816307c3780c/data_sync/mvds.md) | Clarifications on spec, RFC-2119 and message identifiers. |
+| [0.5.0](https://github.com/status-im/bigbrother-specs/blob/6d3d0771b46941791eabe9263f80cbaeaf3d1148/data_sync/mvds.md) | Initial Release |
```

Comment:
archived repo, does it make sense to link there?

Reply:
I think it makes sense to still be able to easily see the history.